src/Api.php

```diff
@@ -128,6 +128,8 @@ public function __construct(
 
         if (!isset($http_client)) {
             $http_client = new GClient();
+            $http_client->setDefaultOption('timeout', 30);
```

Comment:
That's the default configuration. But you can put your own Gclient with custom timeout if you want. I add it into the documentation